docs/embedding/modular-embedding.md

```diff
@@ -28,18 +31,18 @@ Currently you can choose to embed:
 
 You can also follow the setup guide directly in Metabase in **Admin settings > Embedding > Setup guide**. We're recording the steps here for convenience.
 
-### 1. Enable Embedded Analytics JS
+### 1. Enable modular embedding
 
-1. In Metabase, go to **Admin settings > Embedding > Modular embedding**.
-2. Toggle on **Embedded Analytics JS**.
+1. In Metabase, go to **Admin settings > Embedding**.
+2. Toggle on **Enable modular embedding**.
 3. Under **Cross-Origin Resource Sharing (CORS)**, add the URLs of the websites where you want to embed Metabase (such as `https://*.example.com`). For testing embeds, you can use `localhost` which is always included in CORS policy.
 4. If you are embedding Metabase components in a domain that's different from your Metabase's domain (including when you're testing the app locally but use Metabase Cloud), go to **Admin settings > Embedding > Security** and set **SameSite cookie** to **None**.
 
 ### 2. Create a new embed
 
-1. In Metabase, go to **Admin > Embedding > Modular embedding**, and select **New embed** next to **Embedded analytics JS**.
+1. In Metabase, open a command palette with Ctrl/Cmd+K, type "New embed" and select "new embed" command. This will open the interactive izard that you can use to set up your embed.
```

Comment:
```suggestion
1. In Metabase, open a command palette with Ctrl/Cmd+K, type "New embed" and select "new embed" command. This will open the interactive wizard that you can use to set up your embed.
```